python/datasources/age_adjust_cdc_restricted.py

```diff
@@ -116,146 +146,123 @@ def merge_age_adjusted(df, age_adjusted_df):
        df: a dataframe with covid date without age adjusted numbers
        age_adjusted_df: a dataframe with age adjusted covid numbers"""
 
-    merge_cols = [std_col.STATE_FIPS_COL, std_col.STATE_NAME_COL]
-    merge_cols.extend(std_col.RACE_COLUMNS)
+    merge_cols = [std_col.STATE_FIPS_COL, std_col.STATE_NAME_COL,
+                  std_col.RACE_CATEGORY_ID_COL]
 
     df = df.reset_index(drop=True)
     age_adjusted_df = age_adjusted_df.reset_index(drop=True)
 
     return pd.merge(df, age_adjusted_df, how='left', on=merge_cols)
 
 
-def get_expected_hosps(race_and_age_df, population_df):
-    """Calculates the age adjusted expected hospitalizations of each racial group.
-       I made this function to break up the age adjustment into smaller, more easily testable pieces.
+def get_expected_col(race_and_age_df, population_df, expected_col, raw_number_col):
+    """Calculates the age adjusted expected deaths of each racial group.
```

Comment:
this is multi-purpose now right? So should the doc string be updated to say like "deaths or hospitalizations" instead of just deaths?

Reply:
Yup, good catch, thanks